### PR TITLE
feat: integration with feathers v3 and feathers-cli

### PR DESCRIPTION
**Pull request by subhadeepdas91**
On selection of "feathers" option during server selection a feather 3 project is created which is compatible with feathers-cli, further feathers-cli can be used to generated components like services, authentication etc.

Feathers config and test directories are contained within server directory

### CODE REVIEW

File: template/_package.json [hunk 1]
```diff
@@ -53,7 +66,8 @@
     "@nuxtjs/pwa": "^2.6.0"<% } %><% if (ui === 'tachyons') { %>,
     "tachyons": "^4.11.1"<% } %>
   },
-  "devDependencies": {
+  "devDependencies": { <% if (server === 'feathers') { %>
+    "@feathersjs/cli": "^3.8.5",<% } %>
```
Comment (clarkdo) [inline template/_package.json]: "nodemon" is better to be first

File: template/_package.json [hunk 2]
```diff
@@ -3,8 +3,14 @@
...
-  "private": true,
-  "scripts": {<% if (server === 'adonis') { %>
+  "private": true, <% if (server === 'feathers') { %>
+  "directories": {
+    "lib": "server",
+    "config": "server/config",
+    "test": "server/test"
+  }, <% } %>
+  "scripts": {<% if (server === 'feathers') { %>
+    "feathers": "feathers",<% } %><% if (server === 'adonis') { %>
```
Comment (clarkdo) [inline template/_package.json]: Should be `else if`

Comment (subhadeepdas91) [reply template/_package.json]: placing `else if` causes non execution of `else` block immediately after `if (server === 'adonis')`
resulting to non generation of `dev, build, start, generate` tasks

should I place the `feathers` task as last task (after `test`)?

File: template/_package.json (hunk 1 above)
Comment (subhadeepdas91) [reply template/_package.json]: Fixed in commit 9f9aa8e394dfb948e6ea533e402a9ec32e1b7534

File: template/_package.json (hunk 2 above)
Comment (clarkdo) [reply template/_package.json]: Oh, I gotcha. Yes, putting `if (server === 'feathers')` after `dev, build, start, generate` makes more sense

Comment (subhadeepdas91) [reply template/_package.json]: Fixed in commit 1910cb0da92dbcbb76618c19b61afb78bc0682d2

**Comment by clarkdo** [comment]
@subhadeepdas91 Could you please update the snap, I think it's ready to merge now.

File: template/_package.json [hunk 3]
```diff
@@ -3,7 +3,12 @@
   "version": "1.0.0",
   "description": "<%= description %>",
   "author": "<%= author %>",
-  "private": true,
+  "private": true, <% if (server === 'feathers') { %>
```
Comment (clarkdo) [inline template/_package.json]: The space after `,` can be removed.

Comment (subhadeepdas91) [reply template/_package.json]: Unnecessary space removed on commit fd0f9c4eb844e0b26483b6bc9da6f8dd3cc410ee

**Comment by clarkdo** [comment]
As a result of team decision, server frameworks have been removed https://github.com/nuxt/create-nuxt-app/pull/501

**Pull request closed without merging**